src/modules/nfts/NftClient.ts

```diff
@@ -20,33 +20,31 @@ import {
 } from './operations';
 
 export class NftClient extends ModuleClient {
-  async findNftByMint(mint: PublicKey): Promise<Nft> {
```

Comment:
I'm not opposed to removing `async` when it's unnecessary but I usually like to explicitly mark all asynchronous methods as `async` anyway so it's super obvious to developers that these functions will return a Promise rather than having to analyse the return type (if there's even one in the first place).

What do you think about that reasoning? If you disagree, I'm happy to remove them as long as we stay consistent with that decision.

Reply:
It's obvious already since the return type is _Promise_ and arguably people taking the step to jump to the declaration should also see that return type.

I've been digging for pros/cons of using `async` when no `await` is inside the function:
https://stackoverflow.com/a/44114047/97443 agrees with what I said and in this thread the other cited reason to use an `async` without an `await` is when the function could potentially throw a synchronous error.

In the end in the above cases it's just cosmetic as long as `v8` does not end up wrapping each result in a promise and unwrapping on the other end (as I thought it did at some point when it sees an `async`).

Whichever way we go, I agree that it's important to be consistent.